w3-account.md

```diff
@@ -76,68 +108,78 @@ Alice -->> App: alice@web.mail
+> ⚠️ [w3up] implementation currently does not support this signature type.
 
-The signer MUST derive the "DKIM payload" from the received message according to the [RFC6376] specification and encode it in UTF-8 encoding. The resulting bytes MUST be encoded as a [Nonstandard `VarSig` signature] with the `alg` parameter set to `"DKIM"`.
+The [`did:mailto`] principal MAY issue a delegation signed using a DomainKeys Identified Mail ([DKIM]) signature.
 
-##### Authorization Payload
+This signature MUST be generated by sending a message from the [account] email address corresponding to the issuer [`did:mailto`] with the `Subject` header set to the [authorization payload].
 
-> ℹ️ Note that the UCAN standard signing payload would results in large signatures. Therefore, we propose an alternative payload format.
+The signer MUST derive the "DKIM payload" from the received message according to the [RFC6376] specification and encode it in UTF-8 encoding. Resulting bytes MUST be encoded as a [Nonstandard `VarSig` signature] with the `alg` parameter set to `"DKIM"`.
 
-The UCAN data model MUST follow the structure specified in the [UCAN-IPLD Schema], but the `s` field should be omitted. The IPLD [link] of the data model must be derived and formatted according to the [ABNF] definition below, where `cid` refers to the derived link:
+#### Authorization Payload
+
+The [UCAN] data model for the desired delegation issued by [`did:mailto`] MUST be structured per [UCAN-IPLD Schema] specification, except for the `s` field, which MUST be omitted.
+
+The IPLD [link] of constructed data model MUST be derived and used the `cid` when formatting according payload according to the following [ABNF] definition.
 
 ```abnf
 auth := "I am signing ipfs://" cid "to grant access to this account"
 cid  := z[a-km-zA-HJ-NP-Z1-9]+
 ```
 
-#### Authorization Session Signature
+### Attestation Signature
+
+Delegation MAY be authorized through an interactive email flow where [account] holder is emailed request to approve an authorization that gives an agent access to specific set of capabilities. If user approves by clicking embedded link signed [attestation] is issued that confirms that delegation has been authorized through an interactive flow.
+
+In this scenario delegation issued by the [`did:mailto`] identifier MAY be signed using _attestation signature_ type. This signature alone MUST NOT be considered as a valid authorization. Delegation signed with _attestation signature_ MUST be accompanied with [UCAN attestation] issued by the trusted [authority].
+
+If delegation is signed with _attestation signature_, but is not accompanied with [UCAN attestation] from trusted [authority] it MUST be considered invalid. In this scenario implementer MAY initiate interactive verification flow and issue [UCAN attestation] retroactively instead of denying service.
```

Comment:
```suggestion
If delegation is signed with an _attestation signature_, but is not accompanied with a [UCAN attestation] from a trusted [authority] it MUST be considered invalid. In this scenario the implementer MAY initiate an interactive verification flow and issue the [UCAN attestation] retroactively instead of denying service.
```